AGAudioPlayer/AGAudioPlayerUpNextQueue.m

```diff
@@ -17,18 +17,21 @@ @interface AGAudioPlayerUpNextQueue ()
 
 @implementation AGAudioPlayerUpNextQueue
 
+- (void)_commonInit {
```

Comment:
I originally used a dispatch queue for locking, but that was dumb. This change is left over from that, but I figure it's nice having a common init here anyway.